contracts/ERC20MultiDelegate.sol

```diff
@@ -108,15 +117,22 @@ contract ERC20MultiDelegate is ERC1155, Ownable {
         for (
             uint transferIndex = 0;
             transferIndex < Math.max(sourcesLength, targetsLength);
```

Comment:
IIRC one of the gas optimisation suggestions was to calculate the `max` outside the loop.

Reply:
moved it